content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -141,6 +141,80 @@ limit.
 On Linux, resource allocations for Pod level control groups (cgroups) are based on the effective Pod
 request and limit, the same as the scheduler.
 
+### Adopting sidecar containers
+
+This section is relevant for people adopting sidecar containers for their workloads.
+
+The sidecar containers feature is in beta state starting Kubernetes version 1.29 and is enabled by default.
+Some clusters may have this feature disabled or have software installed that is incompatible with the feature.
+Here are the considerations and troubleshooting steps that one can take while adopting sidecar containers for their workload.
+
+**1. Ensure the feature gate is enabled.**
+
+As a very first step, make sure that both - API server and Nodes were upgraded to the version 1.29+.
+The feature will break on clusters where Nodes are running earlier versions where it is not enabled.
+
+The feature gate enablement should also be validated for both - API server AND Nodes.
+
+Once of the ways to check the feature gate enablement is running a command like this:
+
+- For API Server 
+  `kubectl get --raw /metrics | grep kubernetes_feature_enabled | grep SidecarContainers`
+- For the individual node: 
+  `kubectl get --raw /api/v1/nodes/gke-cluster-device-p-almost-real-tpu--122d9b42-tnb5/proxy/metrics | grep kubernetes_feature_enabled | grep SidecarContainers`
+
+If you see something like: `kubernetes_feature_enabled{name="SidecarContainers",stage="BETA"} 1`,
+it means that the feature is enabled.
+
+**2. Check for 3rd party tooling and mutating webhooks**
+
+If trying the feature you experience issues, it may be an indication that one of the
+3rd party tools or mutating webhooks are broken.
+
+The Sidecar feature adds a new field that is define in Kubernetes 1.28 API.
+Some tools and mutating webhooks might have been build with the earlier version of Kubernetes API.
+If tools pass the unknown fields as-is or mutating webhooks are using the recommended `Patch` method to mutate Pods, it will not be a problem.
+
+However there are tooling that will strip out unknown fields and must be recompiled with the fresh version of Kubernetes API.
+
+The easy way to validate if this is a root cause of a problem is to use `kiubectl describe pod` command with your Pod.
+If any tooling stripped out the new field (`restartPolicy:Always`), you will not find see it in the command output.
```

Comment:
```suggestion
If any tools stripped out the new field (`restartPolicy:Always`), you will not find see it in the command output.
```

Reply:
Even better: avoid “new” here. We like [timeless documentation](https://developers.google.com/style/timeless-documentation).